statvar_imports/cdc/social_vulnerability_index/test_data/sample_output/SVI_2022_US_county_output_schema_mcf.mcf

```diff
@@ -0,0 +1,47 @@
+Node, dcid,socialVulnerabilityIndex
```

Comment:
Pls change to the locally generated mcf file using output_path.

The sheet downloade as csv is not in MCF format as it has extra lines with `,,` , double quites for `"name: "..."",,`